pkg/operator/resource/resourcehelper/resource_helpers.go

```diff
@@ -0,0 +1,74 @@
+package resourcehelper
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/scheme"
+
+	"github.com/openshift/api"
+)
+
+var (
+	openshiftScheme = runtime.NewScheme()
+)
+
+func init() {
+	if err := api.Install(openshiftScheme); err != nil {
+		panic(err)
+	}
+}
+
+// ResourceSelectorWithNamespaceForCLI generates a string that can be copy/pasted for use with oc get that includes
+// specifying the namespace with the -n option.
+func ResourceSelectorWithNamespaceForCLI(obj runtime.Object) string {
+	groupVersionKind := GuessObjectGroupVersionKind(obj)
+	kind := groupVersionKind.Kind
+	group := groupVersionKind.Group
+	var name, namespace string
+	accessor, err := meta.Accessor(obj)
+	if err != nil {
+		name = "<unknown>"
+		namespace = "<unknown>"
+	}
+	name = accessor.GetName()
+	namespace = accessor.GetNamespace()
+	if len(group) > 0 {
+		group = "." + group
+	}
+	if len(namespace) > 0 {
+		namespace = " -n " + namespace
+	}
+	return kind + group + "/" + name + namespace
+}
+
+// ResourceSelectorForCLI generates a string that can be copy/pasted for use with oc get.
+func ResourceSelectorForCLI(obj runtime.Object) string {
+	groupVersionKind := GuessObjectGroupVersionKind(obj)
+	kind := groupVersionKind.Kind
+	group := groupVersionKind.Group
+	var name string
+	accessor, err := meta.Accessor(obj)
+	if err != nil {
+		name = "<unknown>"
+	}
+	name = accessor.GetName()
+	if len(group) > 0 {
+		group = "." + group
+	}
+	return kind + group + "/" + name
+}
+
+// guessObjectGroupVersionKind returns a human name for the passed runtime object.
```

Comment:
human readable

Reply:
It returns "Bob", "John", "Stacy", etc... 
Just kidding :wink: . Fixed.